MapboxMaps.podspec

```diff
@@ -12,19 +12,20 @@ Pod::Spec.new do |m|
   m.author = { 'Mapbox' => 'mobile@mapbox.com' }
   m.social_media_url = 'https://twitter.com/mapbox'
   m.documentation_url = 'https://docs.mapbox.com/ios/beta/maps/api-reference/'
-
-  m.source = { http: "https://api.mapbox.com/downloads/v2/mobile-maps-ios/releases/ios/packages/#{maps_version.to_s}/MapboxMaps.xcframework.zip" }
-  m.vendored_frameworks = 'MapboxMaps.xcframework'
-
+  
+  m.source = { :git => 'https://github.com/mapbox/mapbox-maps-ios.git', :tag => 'v10.0.0-beta.13' }
   m.platform = :ios
   m.ios.deployment_target = '11.0'
   m.swift_version = '5.3'
   m.requires_arc = true
   m.pod_target_xcconfig = { 'EXCLUDED_ARCHS[sdk=iphonesimulator*]' => 'arm64' }
```

Comment:
Is this workaround still necessary? I don’t think any of the dependencies excludes ARM64 when built from source.

/ref https://github.com/mapbox/mapbox-navigation-ios/issues/2672#issuecomment-782549228

Reply:
No with a build from source it is not necessary anymore